### ISSUE TITLE
ott.tools.soft_sort.ranks doc mismatch with function signature

### ISSUE THREAD

**Issue by pierreablin** [ott-jax/ott#395]
The doc https://ott-jax.readthedocs.io/en/latest/_autosummary/ott.tools.soft_sort.ranks.html#ott.tools.soft_sort.ranks mentions a `topk` parameter which raises an error

```python
soft_sort.ranks(x, topk=5)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [14], in <cell line: 1>()
----> 1 soft_sort.ranks(x, topk=5)

File /opt/homebrew/anaconda3/lib/python3.9/site-packages/ott/tools/soft_sort.py:214, in ranks(inputs, axis, num_targets, **kwargs)
    185 def ranks(
    186     inputs: jnp.ndarray,
    187     axis: int = -1,
    188     num_targets: Optional[int] = None,
    189     **kwargs: Any,
    190 ) -> jnp.ndarray:
    191   r"""Apply the soft trank operator on input tensor.
    192 
    193   Args:
   (...)
    212     A jnp.ndarray<float> of the same shape as inputs, with the ranks.
    213   """
--> 214   return apply_on_axis(_ranks, inputs, axis, num_targets, **kwargs)

File /opt/homebrew/anaconda3/lib/python3.9/site-packages/ott/tools/soft_sort.py:105, in apply_on_axis(op, inputs, axis, *args, **kwargs)
    102 inputs = jnp.transpose(inputs, permutation)
    104 batch_fn = jax.vmap(op_inner, in_axes=(0,) + (None,) * len(args))
--> 105 result = batch_fn(jnp.reshape(inputs, (-1, num_points)), *args)
    106 shrink = len(axis)
    107 result = jnp.reshape(result, inputs.shape[:-shrink] + result.shape[-1:])

    [... skipping hidden 3 frame]

File /opt/homebrew/anaconda3/lib/python3.9/site-packages/ott/tools/soft_sort.py:180, in _ranks(inputs, num_targets, **kwargs)
    178 a = jnp.ones((num_points,)) / num_points
    179 b = jnp.ones((num_targets,)) / num_targets
--> 180 ot = transport_for_sort(inputs, a, b, **kwargs)
    181 out = 1.0 / a * ot.apply(jnp.arange(num_targets), axis=1)
    182 return jnp.reshape(out, inputs.shape)

File /opt/homebrew/anaconda3/lib/python3.9/site-packages/ott/tools/soft_sort.py:75, in transport_for_sort(inputs, weights, target_weights, squashing_fun, epsilon, **kwargs)
     72 geom = pointcloud.PointCloud(x, y, epsilon=epsilon)
     73 prob = linear_problem.LinearProblem(geom, a=a, b=b)
---> 75 solver = sinkhorn.Sinkhorn(**kwargs)
     77 return solver(prob)

TypeError: __init__() got an unexpected keyword argument 'topk'
```

**Comment by marcocuturi** [comment]
yup, this is a copy pasting bug in a recent PR, am on it!

**Comment by marcocuturi** [comment]
closed by #396
